source/puppet/4.1/reference/lang_facts_and_builtin_vars.markdown

```diff
@@ -79,11 +78,11 @@ All facts appear in Puppet as [top-scope variables][topscope]. They can be acces
 
 Example, with the osfamily fact:
```

Comment:
maybe: "Example, using the `family` fact, which aggregates all Red Hat-derived Linux variants into one 'redhat' value."